src/components/walletSelectButton/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { useToggleVisibility } from '@pivanov/use-toggle-visibility';
+import { useLocation } from 'react-router-dom';
+
+import { Icon } from '@components/icon';
+import { ModalWalletSelect } from '@components/modals/modalWalletSelect';
+import { useStoreWallet } from '@stores';
+import { cn } from '@utils/helpers';
+
+const WalletSelectButton = () => {
+
+  const { pathname } = useLocation();
+  const isHomePage = pathname === '/';
+
+  const disconnectWallet = useStoreWallet?.use?.actions?.()?.disconnect;
+  const accounts = useStoreWallet?.use?.accounts?.();
+
+  const [
+    WalletSelectModal,
+    toggleVisibility,
+  ] = useToggleVisibility(ModalWalletSelect);
+
+  if (isHomePage) {
+    return null;
+  }
+
+  return (
+    <>
+      {
+        accounts.length > 0 ? (
+          <button
+            type="button"
+            onClick={disconnectWallet}
+            className={cn(
+              'flex min-h-10 items-center px-6 py-1',
```

Comment:
why the min-h-10
i would remove the min-h-10 / change to py-2 => same result without hardcoding

Reply:
done